### PR TITLE
GFM-86 - Use section headers as titles

### PR DESCRIPTION
**Pull request by joefitter**
* Added section headers to each page to replace label title.
* Added labels to first input
* Added custom sub-title to post step

### CODE REVIEW

File: apps/gro/fields/when.js [hunk 1]
```diff
@@ -1,20 +1,14 @@
 'use strict';
 
 module.exports = {
-  'when-date': {
-    legend: 'fields.when-date.legend',
-    hint: 'fields.when-date.hint'
-  },
+  'when-date': {},
```
Comment (JoeChapman) [inline apps/gro/fields/when.js]: doesn't it need a hint?

File: apps/gro/translations/src/en/fields.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "about-radio": {
+    "legend": "What would you like to contact us about?",
```
Comment (JoeChapman) [inline apps/gro/translations/src/en/fields.json]: translation

File: apps/gro/translations/src/en/fields.json [hunk 3]
```diff
@@ -22,6 +23,7 @@
     }
   },
   "type-radio": {
+    "legend": "What type of certificate did you order?",
```
Comment (JoeChapman) [inline apps/gro/translations/src/en/fields.json]: translation

File: apps/gro/translations/src/en/fields.json [hunk 4]
```diff
@@ -40,25 +42,32 @@
       }
     }
   },
+  "details-text": {
+    "label": "Provide details of your complaint"
```
Comment (JoeChapman) [inline apps/gro/translations/src/en/fields.json]: translation

File: apps/gro/translations/src/en/fields.json [hunk 5]
```diff
@@ -40,25 +42,32 @@
...
+  },
   "existing-radio": {
     "legend": "Is your complaint about an existing order?"
   },
   "previous-radio": {
     "legend": "Have you already complained about this issue?"
   },
   "person-text": {
-    "label": "Full name"
+    "label": "Whose name, including any initials, is on the certificate you received?"
```
Comment (JoeChapman) [inline apps/gro/translations/src/en/fields.json]: translation

File: apps/gro/fields/when.js (hunk 1 above)
Comment (joefitter) [inline apps/gro/fields/when.js]: implicit

**Comment by bendiggle** [comment]
Does this all have to be in one commit? Or could it broken down to match your description in the comments?

**Comment by joefitter** [comment]
It _could_ if we are being pedantic ;)

**Comment by joefitter** [comment]
Personally I don't think this is too much for a single commit - I know a lot of files were amended but in most cases its a single line change and all related to the same small task. If the ticket had a few subtasks then I'd factor it so each commit was a subtask, but I think splitting this down would be overkill and not particularly useful

**Pull request merged**